pkg/build/builder/docker.go

```diff
@@ -262,6 +262,63 @@ func (d *DockerBuilder) copySecrets(secrets []buildapiv1.SecretBuildSource, buil
 	return nil
 }
 
+// copyConfigs copies all files from the directory where the secret is
+// mounted in the builder pod to a directory where the is the Dockerfile, so
+// users can ADD or COPY the files inside their Dockerfile.
+func (d *DockerBuilder) copyConfigs(configs []buildapiv1.ConfigMapBuildSource, buildDir string) error {
+	for _, c := range configs {
```

Comment:
A lot is copy-pasted here...opportunity to refactor.

Reply:
agreed ... I would go ahead and suggest you give that a go